YvesIraguha/lost-and-found-app
# chore(tailwindcss): setup tailwindcss

#### What does this PR do?

- Setup tailwind css 

#### Description of Task to be completed? 

- Setup tailwind css
- Apply tailwind predefined classes to welcoming text

#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch `ch-setting-up-tailwindcss`

- Then run **`npm install`** to install all dependencies

- Run **`npm start`** to start the server

- In your default browser navigate to **`http://localhost:3000`**

> You should see a Rendered welcoming message with tailwind css classes applied

#### Any background context you want to provide?

- N/A
